grabbit/src/main/groovy/com/twcable/grabbit/spring/batch/repository/JcrGrabbitExecutionContextDao.groovy

```diff
@@ -42,7 +40,7 @@ import static org.apache.sling.api.resource.ResourceUtil.getOrCreateResource
  */
 @CompileStatic
 @Slf4j
-class JcrExecutionContextDao extends AbstractJcrDao implements ExecutionContextDao {
+class JcrGrabbitExecutionContextDao extends AbstractJcrDao implements GrabbitExecutionContextDao {
```

Comment:
Renamed to something like this ... I liked the idea of having `Jcr` somewhere in the name indicating what is the implementation made for.